games-engines/dragengine/dragengine-1.2.recipe

```diff
@@ -0,0 +1,108 @@
+SUMMARY="Drag[en]gine Game Engine"
+DESCRIPTION="Provides the game engine and launchers required to run games and other \
+projects based on the Drag[en]gine Game Engine. Games do not compile directly against \
+the game engine. Instead there is one shared installation on the system relying on \
+launchers to handle all the initialization and maintenance work.
+
+This package provides one launcher.
+
+'delauncher-console' is a console based launcher. It is the most basic launcher just \
+able to run the game without providing any additional tools. It can be used to run \
+games without overhead or in headless mode for server operation."
+HOMEPAGE="
+	https://dragondreams.ch
+	https://github.com/LordOfDragons/dragengine
+	"
+COPYRIGHT="2020-2020 Dragon Dreams"
+LICENSE="
+	GNU GPL v3
+	GNU LGPL v3
+	"
+REVISION="1"
+srcGitRev="1.2-1-haiku"
+SOURCE_URI="https://github.com/LordOfDragons/dragengine/archive/v${srcGitRev}.tar.gz"
+CHECKSUM_SHA256="fade8d3fac5ab098f72f686ea59c81e01a33bbc08070fc957eaa3a459f9d7963"
+SOURCE_FILENAME="dragengine-${srcGitRev}.tar.gz"
+SOURCE_DIR="dragengine-${srcGitRev}"
```

Comment:
You can remove the "{}" here also
```diff
-${srcGitRev}
+$srcGitRev
```

Reply:
Fixed them. That should be the last of the ${} pairs.